pkg/ctl/install/install.go.orig

```diff
@@ -0,0 +1,657 @@
+package install
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"time"
+
+<<<<<<< HEAD
```

Comment:
Is this `.orig` file an accidental addition? This looks like the output of git during some conflict resolution, so I'm guessing we don't want it here, do we?

Reply:
upssss thanks!